### ISSUE TITLE
Provide a possibility to download old-orthography Russian plays in new orthography

### ISSUE THREAD

**Issue by lehkost** [dracor-org/dracor-api#113]
We now start to feature Russian plays in old orthography, the first one being [Artsybashev's »Revnost'«](https://dracor.org/rus/artsybashev-revnost). We also just released a Python package which does the conversion: [prereform2modern](https://pypi.org/project/prereform2modern/).

@DanilSko suggested to have a cron job that once a day converts the original TEI of old-orthography plays to an additional version in new orthography and make it available through our API. (Doing this on-the-fly when requested will probably be far too costly).

(As inspiration, Deutsches Textarchiv (@cthomasdta) also provides "normalised" (= modernised) text output, e.g. [here](http://www.deutschestextarchiv.de/book/download_normxml/collin_coriolan_1804) (see file header for details).)

**Comment by cmil** [comment]
I have the following questions:

1. How do we identify documents using the old orthography? Since pre-reform Russian does not seem to have it's own language code, we cannot use the `xml:lang` attribute.
2. Would it be safe to pass an entire TEI document to `prereform2modern` for transliteration or would each text node have to be transliterated separately?

I think, we better create a transliterated version whenever a document changes using a trigger instead of running a cronjob.

It would also be very helpful if the `translit` command line script provided by `prereform2modern` would accept a file path as an argument and would return the transliterated document as output. Given that the library is already capable of adding markup that would be a logical next step.

**Comment by lehkost** [comment]
Re 1. True, missing lang code for pre-reform Russian is a problem. A simple solution would be to look for deprecated letters like "ѣ" in the documents. Just did a grep on all files and it works. 😊 Yet it would probably be better to add markup so we can intrinsically tell apart an original xml file from a converted one.

Re 2. I think it's possible to pass an entire TEI doc through the converter. (@DanilSko @nevmenandr?)

Re 3. Agree with the trigger upon changes instead of cronjob.

**Comment by cthomasdta** [comment]
Yes, store this kind of info properly where it belongs in the `teiHeader`. I already wanted to warn you that the header in the example file you used is not best practice in any sense. We had to dumb it down for Voyant ingest.

**Comment by lehkost** [comment]
Thanks for the heads-up, Christian. Then the question would be how to store the info in case of a play in pre-reform orthography next to the general languade code in the root element (```<TEI xml:lang="rus">```).

(RusDraCor example file in TEI [here](https://dracor.org/api/corpora/rus/play/artsybashev-revnost/tei).)

**Comment by cthomasdta** [comment]
Sure you could customise `@xml:lang` and use a `#pointer`? But maybe I miss your point, will have to check again next week and look at your example. No work to be done outside in the green on a mobile phone...
